packages/tabs/src/Tabs/Tabs.tsx

```diff
@@ -195,25 +154,27 @@ function Tabs(props: AccessibleTabsProps) {
   });
 
   return (
-    <div {...rest} className={className}>
-      {/* render the portaled contents */}
-      {renderedTabs}
-
-      <div className={tabContainerStyle}>
-        {/* renderedTabs portals the tab title into this element */}
-        <div
-          className={cx(listStyle, modeColors[theme].underlineColor)}
-          role="tablist"
-          ref={setTabNode}
-          aria-orientation="horizontal"
-          {...accessibilityProps}
-        />
-        <div className={inlineChildrenWrapperStyle}>{inlineChildren}</div>
+    <LeafyGreenProvider baseFontSize={baseFontSize === 16 ? 16 : 14}>
```

Comment:
Not related to this PR, but maybe we also pass darkmode here so children can inherit it? Or this could be for a separate PR.

Reply:
We're passing through children but agreed that this could be updated in another PR